chart/zipkin-server/templates/deployment.yaml

```diff
@@ -48,7 +48,15 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           env:
             - name: STORAGE_METHOD
```

Comment:
Regarding to https://github.com/openzipkin/zipkin/tree/master/zipkin-server  
it should be STORAGE_TYPE

Reply:
well, it's merged now. Feel free to open a PR to fix this, though (I'm not an official zipkin dev)